resolwe_bio/processes/workflows/rnaseq.yml

```diff
@@ -0,0 +1,203 @@
+- slug: workflow-rnaseq-single
+  category: workflow:primary
+  name: Trimmomatic filtering followed by HISAT2 aligner and HTSeq-count-normalized quantifier
+  data_name: "Pipeline ({{ reads|sample_name|default('?') }})"
+  requirements:
+    expression-engine: jinja
+  version: 0.0.1
+  type: data:workflow:rnaseq:htseq
+  input:
+    - name: reads
+      label: Input reads
+      type: data:reads:fastq:single
+    - name: genome
+      label: Genome
+      type: data:genome:fasta
+    - name: annotation
+      label: Annotation (GTF)
+      type: data:annotation:gtf
+    - name: adapters
+      label: Adapter sequences (FASTA)
+      type: data:seq:nucleotide
+      required: false
+    - name: illuminaclip
+      label: Illumincaclip
+      group:
+        - name: seed_mismatches
+          label: Seed mismatches
+          type: basic:integer
+          required: false
+          description: >
+             Specifies the maximum mismatch count which will still allow a full match to be performed.
+          default: 2
```

Comment:
If you set a default value, there is no need for a `required: false` since the field value is always given.